src/screens/add-okr/add-okr.tsx

```diff
@@ -0,0 +1,209 @@
+import React from 'react';
+import { RouteProp, useNavigation, useRoute } from '@react-navigation/native';
+import { ScrollView, View } from 'react-native';
+import { SafeAreaView } from 'react-native-safe-area-context';
+import { NativeStackNavigationProp } from '@react-navigation/native-stack';
+import { Formik } from 'formik';
+
+import { AppRoute, ButtonMode, HeadingLevel } from 'src/common/enums';
+import {
+  Heading,
+  MainButton,
+  FormInput,
+  FormSelect,
+  Text,
+  AddButton,
+} from 'src/components';
+import {
+  createOKRValidationSchema,
+  createTeamOKRValidationSchema,
+} from 'src/validation-schemas';
+import { defaultAddOKRPayload } from './common';
+import { AppStackParamList } from 'src/common/types';
+import { useAppDispatch, useAppSelector } from 'src/hooks';
+import { okrActions } from 'src/store/okr';
+import { IAddOkr } from 'src/common/types/okr/add-okr.interface';
+import styles from './styles';
+
+const MOCK_TEAM_NAMES = [
+  {
+    label: 'Team 1',
+    value: 'team1',
+  },
+  {
+    label: 'Team 2',
+    value: 'team2',
+  },
+  {
+    label: 'Team 3',
+    value: 'team3',
+  },
+];
+
+const MOCK_PARENT_OBJ = [
+  {
+    label: 'Improve JS Skills',
+    value: 'Improve JS Skills',
+  },
+  {
+    label: 'Improve Team Coop',
+    value: 'Improve Team Coop',
+  },
+];
+
+const MOCK_OBJ_CYCLE = [
+  {
+    label: 'Strategic 2020',
+    value: 'Strategic 2020',
+  },
+  {
+    label: 'Strategic 2021',
+    value: 'Strategic 2021',
+  },
+  {
+    label: 'Strategic 2022',
+    value: 'Strategic 2022',
+  },
+];
+
+type AddOKRNavigateProps = NativeStackNavigationProp<
+  AppStackParamList,
+  AppRoute.ADD_KEY_RESULT
+>;
+
+type AddOKRRouteProps = RouteProp<AppStackParamList, AppRoute.ADD_OKR>;
+
+const AddOKRScreen: React.FC = () => {
+  const dispatch = useAppDispatch();
+  const { currentKeyResults } = useAppSelector(state => state.okr);
```

Comment:
```suggestion
  const { current } = useAppSelector(state => state.okr);
```
We might need to store some additional okr-related data there